src/jit/emitxarch.cpp

```diff
@@ -5324,6 +5334,41 @@ void emitter::emitIns_SIMD_R_R_R(instruction ins, emitAttr attr, regNumber reg,
     }
 }
 
+void emitter::emitIns_SIMD_R_R_I(instruction ins, emitAttr attr, regNumber reg, regNumber reg1, int ival)
+{
+    bool isSSEShift = false;
```

Comment:
Maybe this should be in a separate "isSseShift" function?

Reply:
Yes, I will refactor here.